rules/config.js

```diff
@@ -56,6 +57,14 @@ module.exports = {
     'prefer-const': 2,
     'prefer-arrow-callback': 2,
     'strict': [ 2, 'never' ],
-    'no-const-assign': 2
+    'no-const-assign': 2,
+
+    // Node
+    'node/no-exports-assign': 'error',
+    'node/no-extraneous-require': 'error',
```

Comment:
nice, this should resolve what we talked about earlier as well w/ missing deps in package.json